src/helix-ui/styles/modules/box.less

```diff
@@ -1,21 +1,19 @@
-.hxBox {
-  &-xs {
-    padding: 0.5rem; // 8px
-  }
+.hxBox-xs {
+  padding: 0.5rem; // 8px
+}
 
-  &-sm {
```

Comment:
Explicitly defined the classes, because `&:extend()` in LESS doesn't work with the shorthand definition.